packages/sources/src/styling/variables/zindex.scss

```diff
@@ -1,5 +1,31 @@
-$zindex-navbar: 1000;
-$zindex-dropdown: 1000;
-$zindex-popover: 1060;
-$zindex-tooltip: 1070;
-$zindex-navbar-fixed: 1030;
+// Selectors
+$zx-viewer-grid-sizing-buttons:     -1;
+$zx-viewer-grid-map-panel:          100;
+
+$zx-selectable:                     100;
+
+$zx-datepicker-base:                100;
```

Comment:
Waarom hier `-base` en elders niks? Ik zou zeggen drop die base.

Reply:
Dropping the bass.. un tss un tss un tss un tss